Content.Shared/Clothing/Components/ClothingComponent.cs

```diff
@@ -44,9 +44,21 @@ public sealed class ClothingComponent : Component
     [DataField("sprite")]
     public string? RsiPath;
 
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField("mask")]
+    public ClothingMask Mask = ClothingMask.UniformFull;
+
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField("maleMask")]
+    public ClothingMask MaleMask = ClothingMask.UniformFull;
```

Comment:
shouldnt the defaults here be the male/female/unisex enum?

Reply:
Unsure if I understand what you mean, these are defining a variation of the mask sprite to use (full, top, none)